### ISSUE TITLE
./generate-go.sh: line 13: goimports: command not found

### ISSUE THREAD

**Issue by breeswish** [pingcap/tipb#111]
Following README doesn't work, fail with error:

```
$ make
./generate-go.sh
generate go code...
./generate-go.sh: line 13: goimports: command not found
make: *** [go] Error 127
```

**Comment by breeswish** [comment]
The following command might be required:

```bash
go get golang.org/x/tools/cmd/goimports
```

**Comment by Xuanwo** [comment]
Could be fix after #199 get merged.
